src/components/MaterialUI/DataDisplay/PredefinedElements/ActionModal.test.js

```diff
@@ -66,6 +66,68 @@ describe("ActionModal", () => {
 		expect(component, "when mounted", "to satisfy", expected);
 	});
 
+	it("Renders ActionModal correctly with disabled actions", () => {
```

Comment:
Can you modify the previous test and add `disabled={false}` to the expected action?